man/corsix-th.6.md

```diff
@@ -0,0 +1,56 @@
+% corsix-th(6) | Games Manual
+%
+% 2018-08-13
+
+# NAME
+
+corsix-th - A Theme Hospital engine reimplementation.
+
+# SYNOPSIS
+
+**corsix-th** [--interpreter=*luafile*] [--config-file *configfile*] [--load= *savefile*] [--dump=strings]
+
+# DESCRIPTION
+
```

Comment:
Some sections have an empty line, while other have not, adding an empty line everywhere probably looks best for the two sections above